tests/target/issue_5700.rs

```diff
@@ -0,0 +1,53 @@
+// rustfmt-config: issue_5700.toml
```

Comment:
If you decide to remove the `issue_5700.toml` this could change to:

```suggestion
// rustfmt-max_width: 120
```